front_end/src/components/landing_page/Landing.js

```diff
@@ -71,53 +118,189 @@ class Landing extends Component {
+        NumStars: 5,
+        GuestRating: 9.6,
+        LowPrice: 900
+      };
 
       let dealsArr = [deals1, deals2, deals3];
 
-      let topDealsMarkup = dealsArr.map(city => {
-        if (city != null) {
+      // topDealMarkup is the collection of cards for hotel deals of the week
+      let topDealsMarkup = dealsArr.map(deal => {
+        if (deal != null) {
           return (
-            <Card>
-              <CardActionArea>
+            <Grid item xs={12} sm={4}>
+              <Card className={classes.card}>
+                <CardHeader
+                  className={classes.root}
+                  avatar={
+                    <Avatar aria-label="City" className={classes.avatar}>
+                      {deal.CityAbbr}
+                    </Avatar>
+                  }
+                  title={deal.HotelName}
+                  subheader={deal.CityName}
+                />
                 <CardMedia
-                  className={classes.cardMedia}
-                  image={city.picurl}
-                  title="Weekend Deals"
+                  className={classes.media}
+                  image={deal.Image}
+                  title={deal.HotelName}
                 />
-                <CardContent>
-                  <Typography variant="h6">{city.picurl}</Typography>
-                  <Typography variant="subtitle1">View Deals</Typography>
+                <CardContent className="dealsSpacing">
+                  <Grid container direction="row" justify="space-between">
+                    <Grid>
+                      <h4>Get {deal.DiscountRate}% off!</h4>
+                    </Grid>
+                    <Grid className="newPrice">
+                      Starting from ${" "}
+                      {(
+                        (deal.LowPrice / 100) *
+                        (100 - deal.DiscountRate)
+                      ).toFixed(2)}
+                    </Grid>
+                  </Grid>
+                  <Grid
+                    container
+                    direction="column"
+                    justify="space-between"
+                    alignItems="center"
+                    className="dealsSpacing2"
+                  >
+                    <Grid>
+                      <ReactStars
+                        count={5}
+                        value={deal.NumStars}
+                        size={28}
+                        edit={false}
+                        color2={"#FFD700"}
+                      />
+                    </Grid>
+                    <Grid className="dealsSpacing">
+                      <h5>Guest Rating: {deal.GuestRating}</h5>{" "}
+                    </Grid>
+
+                    <Grid className="dealsSpacing2">
+                      <Button
+                        class="buttonSearch"
+                        primary
+                        onClick={this.handleClickChooseDeal(
+                          deal.HotelName,
+                          deal.Image
```

Comment:
i think you should add deal.DiscountRate in the parameter. We need to apply it through out the checkout process of the chosen hotel

Reply:
It will be part of the data we get from the backend.